lib/handbook/addon/docs/components/schema-block-group-renderer/controller.ts

```diff
@@ -0,0 +1,161 @@
+                    displayText: 'Remember who was in NSync and who was in Backstreet Boys',
+                    schemaBlockGroupKey: 'q3',
+                    index: 8,
+                },
+                {
+                    id: 'SB23',
+                    blockType: 'select-other-option',
+                    displayText: 'Other',
+                    schemaBlockGroupKey: 'q3',
+                    index: 9,
+                },
+                {
+                    id: 'SB24',
+                    blockType: 'short-text-input',
+                    schemaBlockGroupKey: 'q3',
+                    index: 10,
+                },
+            ],
+        },
+        {
+            groupType: 'multi-select-input',
+            blocks: [
+                {
+                    id: 'SB12',
+                    blockType: 'question-label',
+                    displayText: 'I never understood all the hate for:',
+                    schemaBlockGroupKey: 'q4',
+                    index: 11,
+                },
+                {
+                    id: 'SB13',
+                    blockType: 'multi-select-input',
+                    registrationResponseKey: 'page-one_multi-select',
+                    schemaBlockGroupKey: 'q4',
+                    index: 12,
+                },
+            ],
+            optionBlocks: [
+                {
+                    id: 'SB14',
+                    blockType: 'select-input-option',
+                    displayText: 'Nickelback',
+                    schemaBlockGroupKey: 'q4',
+                    index: 13,
+                },
+                {
+                    id: 'SB15',
+                    blockType: 'select-input-option',
+                    displayText: 'Crocs',
+                    schemaBlockGroupKey: 'q4',
+                    index: 14,
+                },
+                {
+                    id: 'SB16',
+                    blockType: 'select-other-option',
+                    displayText: 'Other:',
+                    schemaBlockGroupKey: 'q4',
+                    index: 15,
+                },
+            ],
+        },
+        {
+            groupType: 'contributor-list',
+            blocks: [
+                {
+                    id: 'SB17',
+                    blockType: 'question-label',
+                    displayText: 'Contributors:',
+                    schemaBlockGroupKey: 'q5',
+                    index: 16,
+                },
+                {
+                    id: 'SB18',
+                    blockType: 'contributor-list',
+                    schemaBlockGroupKey: 'q5',
+                    index: 17,
+                },
+            ],
+        },
+    ];
+    schemaChangeset = {
+        'page-one_short-text': '',
+        'page-one_long-text': '',
+        'page-one_single-select-two': '',
+        'page-one_multi-select': [],
+    };
+    groupChangeset = new Changeset(this.schemaChangeset);
```

Comment:
If we rename `schemaChangeset` to `pageRespose`, then this should be `pageChangeset` or `pageResponseChangeset`.